frontend/packages/dev-console/src/components/edit-application/EditApplication.tsx

```diff
@@ -1,113 +1,104 @@
 import * as React from 'react';
 import { Formik } from 'formik';
 import * as _ from 'lodash';
+import { connect } from 'react-redux';
+import * as plugins from '@console/internal/plugins';
+import { getActivePerspective } from '@console/internal/reducers/ui';
+import { RootState } from '@console/internal/redux';
+import { history } from '@console/internal/components/utils';
 import { NormalizedBuilderImages, normalizeBuilderImages } from '../../utils/imagestream-utils';
 import { createOrUpdateResources } from '../import/import-submit-utils';
```

Comment:
To make it more clear that its for git import forms.
```suggestion
import { createOrUpdateResources as createOrUpdateGitResources } from '../import/import-submit-utils';
```